src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -340,4 +348,49 @@ class EntityClient (requestContext: RequestContext, modelSchema: ModelSchema)(im
       }
     }
   }
+
+  def importPFB(pipeline: WithTransformerConcatenation[HttpRequest, Future[HttpResponse]],
+                workspaceNamespace: String, workspaceName: String, pfbRequest: PfbImportRequest): Future[PerRequestMessage] = {
+    def callArrow = {
+      logger.debug("calling arrow")
+      pipeline {
+        Post(avroToRawlsURL, requestContext.request.entity)
+      }
+    }
+
+    def callRawls(arrowResponse: HttpResponse) = {
```

Comment:
nit: this should accept an `HttpEntity`, or a model object - that would make it much more testable/encapsulated